sql/2019/09_Accessibility/09_08.sql

```diff
@@ -0,0 +1,24 @@
+/*
+standard sql
+09_09
+
+10TB
+
+only tested on the sample response body samples
+presence of lang attribute
+this grabs the lang tag hout o fthe html header, and then works to remove the lang=, and "" from the language.
```

Comment:
You can reuse this query: https://discuss.httparchive.org/t/what-are-the-invalid-uses-of-the-lang-attribute/1022/5?u=rviscomi